### ISSUE TITLE
[SUGGESTION] Support Band-In-The-Box paste

### ISSUE THREAD

**Issue by milnak** [jjazzboss/JJazzLab#412]
in Band-in-a-box, if I select all then copy, it pastes to the clipboard in a very specific format, e.g.

```text
[Chords] 
|:a) FMaj7| Ab7| Gm7| C7| FMaj7| Ab7| G7| C7|
|a) FMaj7| Cm7 F7| BbMaj7| Eb7| FMaj7| D7| G7| C7|
|a) FMaj7| Ab7| Gm7| C7| FMaj7| Ab7| G7| C7|
|a) Cm7| F7| BbMaj7| Eb7| FMaj7/C Gm7/C| FMaj7/C Gm7/C| Am7 Dm7| Gm7 C7|
| FMaj7 D7| G7 C7:3X|
[ChordsEnd] 
[Lyrics]
|                     |     A   fog - gy   | day             in  | London town |             Had  me  |       low          |             and had  | me down |
|           I     view  |   the  mor - ning  |            with  a  | - larm |           The British  |    museum        |      had lost   its  |   charm |
|           How  long  |   I won - de - red  |        could this  | thing last |           But the  | age  of mi - ra -  | cles    hadn't       | passed |
|         For  sud -  | den -  ly           |      I saw  you   | there      And through |         fog-gy Lon- | don town   the sun  | was shi - ning      |    ev- 'ry   whe - |
|          - re |  |
[LyricsEnd]
```

it would be really cool if JJazzLab paste could detect this and paste it into the chart (or perhaps have a File > New From Clipboard ?)

Thanks!

**Comment by jjazzboss** [comment]
I understand, and it's not that difficult to implement, but what would be the objective ?  JJazzLab can already import chords from BIAB song files see https://jjazzlab.gitbook.io/user-guide/songs/importing-songs#band-in-a-box-song-files

**Comment by jjazzboss** [comment]
Will be more or less implemented by Issues #432  and #433
